packages/sirius-web/backend/sirius-web-domain/src/main/java/org/eclipse/sirius/web/domain/boundedcontexts/representationdata/RepresentationMetadata.java

```diff
@@ -164,6 +176,13 @@ public Builder kind(String kind) {
             return this;
         }
 
+        public Builder iconURLs(List<String> iconURLs) {
+            this.iconURLs.addAll(Objects.requireNonNull(iconURLs).stream()
```

Comment:
The builder should receive `List<RepresentationIconURL>`, it's up to caller to perform the convert

Reply:
Done